pytest_variables.py

```diff
@@ -14,14 +27,20 @@ def pytest_addoption(parser):
         action='append',
         default=[],
         metavar='path',
-        help='path to test variables JSON file.')
+        help='path to test variables JSON/HJSON/YAML file.')
 
 
 @pytest.fixture(scope='session')
 def variables(request):
-    """Provide test variables from a JSON file"""
+    """Provide test variables from a JSON file or HJSON/YAML files if installed"""
     data = {}
     for path in request.config.getoption('variables'):
+        ext = os.path.splitext(path)[1].replace(".","").lower()
+        try:
+            mod = importlib.import_module(parser_table[ext][0])
+        except ImportError:
```

Comment:
Another thought is that we should default to JSON if the extension is not found in the parser table or is empty/None.